lsql/judge/tests/test_parse.py

```diff
@@ -170,3 +184,10 @@ def test_zip_errors(self):
             zip_path = os.path.join(curr_path, self.ZIP_FOLDER, filename)
             problem = TriggerProblem(zipfile=zip_path)
             self.assertRaises(ValidationError, problem.clean)
+
+        # Discriminant problems
+        for filename in [self.DISCRIMINANT_MISSING_FILES, self.DISCRIMINANT_BAD_STMT,
```

Comment:
Por curiosidad, ¿qué le falta/sobra al ZIP `DISCRIMINANT_BAD_STMT`?

Reply:
min_stmt = 0.

Para cubrir esta línea: (309 de **parse.py**)
`raise ZipFileParsingException('Invalid value in JSON file: "title", "min_stmt" or "max_stmt"')`